js/jquery/plugins/jquery.colorpicker.js

```diff
@@ -53,8 +53,8 @@
 
     // update the colors of the newly created inputs
     $(document).ready(function() {
-      inputs.forEach(function() {
-        $.fn.mColorPicker.setTextColor($(this));
+      inputs.forEach(function(input) {
+        $.fn.mColorPicker.setTextColor(input);
```

Comment:
Warning: argument passed to `setTextColor(element)` method is treated as a jQuery object.
Here the input argument should be wrapped first : 
```suggestion
        $.fn.mColorPicker.setTextColor($(input));
```
Or even better, just wrap the argument of any type in `setTextColor(element)` method : 
```
        $.fn.mColorPicker.setTextColor = function(element) {
          element = $(element);
          // ...
```

Reply:
Forget my warning please : items in `inputs` array are jQuery objects already.